lib/options.py

```diff
@@ -256,7 +256,7 @@ def __init__(self):
         parser.add_argument(
                 "--vardir",
                 dest="vardir",
-                default="var",
+                default=os.environ.get('VARDIR') or '/tmp/t',
```

Comment:
Let's reflect this logic in the help message.

Reply:
Fixed.